src/utils/helper.ts

```diff
@@ -165,82 +164,11 @@ export function toAttributes(doc: any /*: Doc */) {
   return attribs;
 }
 
-export function toAttributeString(attribs: Attribute[]) {
-  if (!attribs.length) return "";
-  return toHtmlSafeString(`(${attribs.join(", ")}) `);
-}
+// export function toAttributeString(attribs: Attribute[]) {
```

Comment:
Can we tidy this up and remove if not needed? Otherwise if it's not used and you don't want to delete it better to leave it uncommented (less confusing for future maintainers).

Reply:
Sure thing! I have removed for now and we always refer back to the original if we ever need it 👍